src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/BaseProjectConfigurationDimensionProvider.cs

```diff
@@ -50,7 +50,30 @@ private async Task<ImmutableArray<string>> GetOrderedPropertyValuesAsync(Unconfi
         {
             Requires.NotNull(project, nameof(project));
 
-            string? propertyValue = await GetPropertyValueAsync(project);
+            ConfiguredProject? configuredProject = await project.GetSuggestedConfiguredProjectAsync();
+
+            Assumes.NotNull(configuredProject);
+
+            return await ProjectAccessor.OpenProjectForReadAsync(configuredProject, evaluatedProject =>
+            {
+                // Need evaluated property to get inherited properties defines in props or targets.
+                return GetOrderedPropertyValues(evaluatedProject);
+            });
+        }
+
+        /// <summary>
+        /// Gets the property values for the dimension.
+        /// </summary>
+        /// <param name="project"><see cref="Project"/>.</param>
+        /// <returns>Collection of values for the dimension.</returns>
+        /// <remarks>
+        /// From <see cref="IProjectConfigurationDimensionsProvider5"/>.
```

Comment:
I don't understand what this remark means.

Reply:
It was meant to point to the usage of the api. Removed.